setup.py

```diff
@@ -76,7 +76,7 @@
     packages=[package for package in find_packages() if package.startswith("stable_baselines3")],
     package_data={"stable_baselines3": ["py.typed", "version.txt"]},
     install_requires=[
-        "gym==0.21",  # Fixed version due to breaking changes in 0.22
+        "gym==0.26.2",
```

Comment:
v0.26.2 is the last release of gym as development on gym has stopped in favor of gymnasium, we don't expect a new version of gym to be released without a critical reason

Reply:
since v0.19, all gym/gymnasium introduced breaking changes (more or less documented), so until it is fully stable again, i prefer to pin the version.
It also doesn't prevent user from trying newer versions if they really want to.